package/src/utilities/add-virtual-import.ts

```diff
@@ -65,6 +66,12 @@ export const addVirtualImport = ({
 	content: string;
 	updateConfig: HookParameters<"astro:config:setup">["updateConfig"];
 }) => {
+	if (name.startsWith("astro:")) {
+		throw new AstroError(
+			"[addVirtualImport] - Virtual import is not allowed to start with 'astro:' because it's an interval prefix.",
```

Comment:
```suggestion
			`Virtual import name prefix can't be "astro:" (for "${name}") because it's reserved for Astro core.`,
			'Update your virtual import name to not use the "astro:" prefix.',
```

Reply:
@florian-lefebvre done and done